test/functional/Jsr292/testng.xml

```diff
@@ -37,6 +37,7 @@
 			<class name="com.ibm.j9.jsr292.MethodTypeTests"/>
 			<class name="com.ibm.j9.jsr292.MethodHandleProxiesTest"/>
 			<class name="com.ibm.j9.jsr292.MethodHandleTest"/>
+			<class name="com.ibm.j9.jsr292.MethodHandleTest11plus"/>
```

Comment:
- I suspect test will fail when testing against SE80 because of missing `com.ibm.j9.jsr292.MethodHandleTest11plus` in SE80 test material.
- Maybe we can try to create a separate `<test>` for new test so that we can avoid to add dummy files.
- If possible, I would prefer not to include jdk version in the test name - MethodHandleTest11plus